readme.md

```diff
@@ -1,4 +1,4 @@
-# ![Polylang](.github/assets/polylang-title.svg)
+# ![Polylang](.github/assets/polylang-logo.svg) [POLYLANG](https://wordpress.org/plugins/polylang/)
```

Comment:
Looks good, but how would screen readers interpret it? Will it says "Polylang" twice?